src/main/java/de/zalando/aruha/nakadi/repository/kafka/KafkaRepository.java

```diff
@@ -99,6 +94,30 @@ public void createTopic(final String topic, final int partitionsNum, final int r
         }
     }
 
+    public boolean topicExists(final String topic) throws NakadiException {
+        return listTopics()
+                .stream()
+                .map(Topic::getName)
+                .anyMatch(t -> t.equals(topic));
+    }
+
+    public boolean areCursorsCorrect(final String topic, final List<Cursor> cursors) {
```

Comment:
Well, it appears it also checks if cursors' partitions exists, so areCursorsOutOfBounds will be misleading. 
So I just renamed to areCursorsValid, don't really see a possibility to have a more specific name.